pkg/kv/kvserver/replica_raft.go

```diff
@@ -1968,6 +1975,7 @@ func (r *Replica) deliverLocalRaftMsgsRaftMuLockedReplicaMuLocked(
 			log.Fatalf(ctx, "unexpected error stepping local raft message [%s]: %v",
 				raft.DescribeMessage(m, raftEntryFormatter), err)
 		}
+		r.mu.raftTracer.MaybeTrace(m)
```

Comment:
nit: put this before the `Step`? To log events that are about to occur. Also, if there is a fatal (unlikely), we will have the last event that caused it in the log.

Reply:
Done